index.js

```diff
@@ -124,6 +124,8 @@ if (dryRun) {
 const cutoffDate = moment().subtract(numDaysOld, 'd').format('YYYY-MM-DD');
 console.log(`Getting issues that haven't been touched since ${cutoffDate}`);
 
+var pickedIssues;
```

Comment:
can't, I'm reassigning it later on. 
I'm assigning a value in one promise, and then printing it out in another, so I need it declared outside of the whole promise chain. 
If there's a better way to pass it around I'm happy to do that.